starter-code/js/index.js

```diff
@@ -11,7 +11,33 @@ function updatePriceByProduct(productPrice, index){
 }
 
 function getTotalPrice() {
```

Comment:
Instead of making a super-complex function, you have some functions to be filled that will make this one lighter. Then you can call functions inside this function.